dist/variables/_variables.scss

```diff
@@ -62,3 +62,5 @@ $font-size-large-1: $font-size-20;
 $font-size-medium: $font-size-16;
 $font-size-regular: $font-size-14;
 $font-size-small: $font-size-12;
+$card-size-default: 132px;
```

Comment:
Ah I see, I had to use the size for the width in the css grid-template-cols style. Since we are changing the grid to allow it to display number of cols based on breakpoint, then this may not be needed anymore